src/applications/vaos/containers/ReasonForAppointmentPage.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  openFormPage,
+  updateFormData,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+} from '../actions/newAppointment.js';
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
+import FormButtons from '../components/FormButtons';
+import { getFormPageInfo } from '../utils/selectors';
+
+const initialSchema = {
+  type: 'object',
+  required: ['reasonForAppointment'],
+  properties: {
+    reasonForAppointment: {
+      type: 'string',
+      enum: ['routine', 'newIssue', 'medicationConcern'],
```

Comment:
Looking in the var-web codebase, it looks like the ids for these are `routine-follow-up`, `new-issue`, and `medication-concern`. I think we should switch to those.